recipes/stats_can/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "stats_can" %}
+{% set version = "2.3.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6ed800301840682a32ee98a9963118d4fb1d65f49abe5a0f73173c8fd4301dba
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python 3
```

Comment:
This should be `python >=3.0` or similar and the same line should be used in host